amical/_cli/api.py

```diff
@@ -74,7 +74,7 @@ def _select_association_file(args):
         date = hdr.get("DATE-OBS", None)
         ins = hdr.get("INSTRUME", None)
 
-        if target is not None or "":
+        if target not in (None or ""):
```

Comment:
Oops, sorry about that
```suggestion
        if target not in (None, ""):
```